### DISCUSSION TITLE
fallthough doesn't catch all

### DISCUSSION THREAD

**Discussion by PlopTheReal** [sveltejs/kit#5423] [Q&A] [answered]
[fallthough].svelte doesn't catch 404 form urls with subsequent path:

`/someInvalidUrl` => ok
`/someInvalidUrl/otherPage` => not ok

How to catch all urls that are giving 404s?

**Comment by benmccann** [comment]
Your post is unclear. Does ok mean it gives a 404 or 200? What's different between those two URLs? It's not clear to me what routes exist in your application

**Reply by PlopTheReal** [reply]
Yes sorry my problem was to catch any subsequent paths. I didn't knew about `[...fallthough].svelte`

**Comment by AntoninLoubiere** [comment, marked as answer]
Maybe you should try to replace `[fallthough].svelte` with `[...fallthough].svelte` in order to catch subsequent paths.
If it's what you meant:
```
routes
| - api
|    | - [a].svelte
|    | - [...b].svelte
```
`[a].svelte` Will only catch paths like `/api/something`, `/api/blabla`... but will not catch sub paths like `/api/test/test`
Whereas `[...b]` will catch all paths that start with `/api/` (in this example paths like `/api/something` will be firstly pass to `[a].svelte` before `[...b].svete`) so `[...b]` will catch paths like `/api/sub/path/test` (If I correctly understood the [docs](https://kit.svelte.dev/docs/routing#advanced-routing-rest-parameters)).

**Reply by PlopTheReal** [reply]
great thanks a lot!
